docs/content/about-iota/iota-wallet-dashboard/getting-started.mdx

```diff
@@ -0,0 +1,45 @@
+---
+description: A guide for getting started with the IOTA Wallet Dashboard including managing staked and vested assets, migrating your Stardust assets and managing your accounts and balances.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet]
+---
+
+# Getting Started with the IOTA Wallet Dashboard
+
+The IOTA Wallet offers an easy-to-use platform for managing your digital assets on the IOTA network.
```

Comment:
Shouldnt this be referring to the Dashboard instead of the wallet?

Reply:
```suggestion
The IOTA Wallet  Dashboard offers an easy-to-use platform for managing your digital assets on the IOTA network.
```